src/apps/cs2cs.cpp

```diff
@@ -94,6 +96,54 @@ using namespace NS_PROJ::metadata;
 using namespace NS_PROJ::util;
 using namespace NS_PROJ::internal;
 
+static std::string degToRoman(double angle, const std::string &pos,
+                       const std::string &neg) {
+    const std::string sign = angle > 0.0 ? pos : neg;
+    angle = fabs(angle);
+    double r = angle;
+    r = floor(r * 3600 + .0005);
+    int sec = (int)fmod(r, 60.);
+    r = floor(r / 60.);
+    int min = (int)fmod(r, 60.);
+    r = floor(r / 60.);
+    int deg = (int)r;
+
+    auto intToRoman = [](int i) {
+        int unit = i % 10;
+        int tenth = i / 10 % 10;
+        int houndr = i / 100 % 10;
+        auto oneDigit = [](int digit, const std::string &one,
+                           const std::string &five, const std::string &ten) {
+            switch (digit) {
+            case (1):
+                return one;
+            case (2):
+                return one + one;
+            case (3):
+                return one + one + one;
+            case (4):
+                return one + five;
+            case (5):
+                return five;
+            case (6):
+                return five + one;
+            case (7):
+                return five + one + one;
+            case (8):
+                return five + one + one + one;
+            case (9):
+                return one + ten;
+            default:
```

Comment:
There's a flaw in the implementation due to the lack of zero. We should either fallback to one in that case, or throw an exception "Sorry, Romans could not express this coordinate"

Reply:
Uhmmm, I do not now if it is a flaw, or a feature. It was implemented in a way that `0` uses `no-symbol`. It would be strange something like `d9'12"M` , o even `d'"S d'"OC` for null island. But at the same time I like it even more, because it shows how roman numerals work (or not work).

Reply:
Forget my comment. It seems I forgot how to use roman numbers... Sorry for the noise